From 93b2ceae845a0d534c1092876d788efdbe08df8b Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 13:12:55 -0400
Subject: [ethernet] i40evf: remove unnecessary break after goto

Message-id: <1409663591-32531-133-git-send-email-sassmann@redhat.com>
Patchwork-id: 90610
O-Subject: [RHEL7.1 PATCH 132/148] i40evf: remove unnecessary break after goto
Bugzilla: 1091127
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Fabian Frederick <fabf@skynet.be>

Signed-off-by: Fabian Frederick <fabf@skynet.be>
Signed-off-by: David S. Miller <davem@davemloft.net>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit 11029d03bf5ddea9726513bd499dc35e398bffa3)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40evf/i40evf_main.c b/drivers/net/ethernet/intel/i40evf/i40evf_main.c
index 1b980fb..ed1eb12 100644
--- a/drivers/net/ethernet/intel/i40evf/i40evf_main.c
+++ b/drivers/net/ethernet/intel/i40evf/i40evf_main.c
@@ -2013,7 +2013,6 @@ static void i40evf_init_task(struct work_struct *work)
   }
   adapter->state = __I40EVF_INIT_VERSION_CHECK;
   goto restart;
-  break;
  case __I40EVF_INIT_VERSION_CHECK:
   if (!i40evf_asq_done(hw)) {
    dev_err(&pdev->dev, "Admin queue command never completed\n");
@@ -2039,7 +2038,6 @@ static void i40evf_init_task(struct work_struct *work)
   }
   adapter->state = __I40EVF_INIT_GET_RESOURCES;
   goto restart;
-  break;
  case __I40EVF_INIT_GET_RESOURCES:
   /* aq msg sent, awaiting reply */
   if (!adapter->vf_res) {
-- 
1.7.1